backend/services/debatevsbot.go

```
@@ -80,76 +50,139 @@ func findLastUserMessage(history []models.Message) models.Message {
-	baseInstruction := "Provide only your own argument in your response without simulating an opponent's dialogue. " +
-		"If the user's input appears unclear or off-topic, ask: 'Could you please clarify your question or provide an opening statement?'"
+	baseInstruction := "Provide only your own argument without simulating an opponent’s dialogue. " +
+		"If the user’s input is unclear, off-topic, or empty, respond with a personality-appropriate clarification request, e.g., for Yoda: 'Clouded, your point is, young one. Clarify, you must.'"
 
-	// If no conversation history exists (or only one message), treat this as the opening statement.
+	// Handle opening statement phase
 	if len(history) == 0 || len(history) == 1 {
-		phaseInstruction := "This is the Opening Statement phase. Introduce the topic, clearly state your stance, and outline the advantages or key points supporting your position."
+		phaseInstruction := "This is the Opening Statement phase. Introduce the topic, clearly state your stance, and outline the advantages or key points supporting your position, using your personality’s rhetorical style and universe ties."
 		return fmt.Sprintf(
 			`You are %s, a %s-level debate bot arguing %s the topic "%s".
```

golangci-lint (1.64.8)

SA5009: Printf format %s reads arg #13, but call has only 12 args

(staticcheck)